src/cli/util.js

```diff
@@ -446,13 +446,17 @@ function formatFiles(context) {
   let changedCache = null;
   if (context.argv["only-changed"]) {
     const cacheDir = findCacheDir({ name: "prettier", create: true });
```

Comment:
Better add `const cacheDir = findCacheDir({ name: "prettier", create: true }) || os.tmp();` here otherwise cache doesn't work

Reply:
If I'm reading this correctly, this would write the cache to a unique temporary directory every time it runs. This would defeat the purpose of the writing the cache since it will not be ready the next time it runs.

Reply:
@g-harel no, if we can't find cache dir, `/tmp/.cache/prettier` will be used and in next run cache was read from this location again, so all works find